llama_hub/microsoft_sharepoint/base.py

```diff
@@ -351,32 +520,62 @@ def get_metadata(filename: str) -> Any:
     def load_data(
         self,
         sharepoint_site_name: str,
-        sharepoint_folder_path: str,
+        sharepoint_folder_path: str = "root",
         recursive: bool = False,
+        include: List[str] = ["documents", "pages"],
+        file_types: List[str] = [],
     ) -> List[Document]:
         """
         Loads the files from the specified folder in the SharePoint site.
 
         Args:
             sharepoint_site_name (str): The name of the SharePoint site.
             sharepoint_folder_path (str): The path of the folder in the SharePoint site.
+                                          If `root`, loads data from the root folder of the
```

Comment:
You should update this in the readme as well. That it'll by default extract the files from root.

Reply:
Updated the readme with the new logic implemented in the latest commit to base